assets/src/components/editing/elements/page_link/PageLinkEditor.tsx

```diff
@@ -38,10 +38,22 @@ export const PageLinkEditor = ({ model, commandContext, attributes, children }:
   const showModal = () =>
     window.oliDispatch(
       modalActions.display(
-        <PageLinkModal
-          model={model}
-          commandContext={commandContext}
-          onDone={({ idref }: Partial<ContentModel.PageLink>) => {
+        <SelectModal
```

Comment:
I refactored all disparate selection models to a single reusable component SelectModal